index.bs

```diff
@@ -643,6 +643,20 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} for the associated {{VRDisplay}}.
 
+# Processing Model # {#proc}
+
+## submitFrame ## {#submitframe-proc}
+
+The are multiple situations where applications won't invoke {{VRDisplay}}.{{submitFrame()}} at the expected rate (the {{VRDisplay}} native framerate). The User Agent MUST intervene to protect the user from motion sickness or consistently inform her about different situations such as errors or navigation between pages. This is the breakdown of the different scenarios:
+
+* The application calls {{VRDisplay}}.{{submitFrame()}} at the expected rate, but only a few frames are dropped. The User Agent MUST re-project the last submitted frame.
+
+* The application calls {{VRDisplay}}.{{submitFrame()}} consistently below the expected framerate to a degree that could make the user feel umconfortable. The User agent SHOULD render a trusted interstitial on top of the application until the application improves performance. If the low framerate persists for a long time the User Agent MUST show a message indicating that the application is too slow to be presented.
```

Comment:
Agree with **terminate presentation** and I think may be the message should guide User how to deal with it or give User a proper terminate way.